### DISCUSSION TITLE
hidesBottomBarWhenPushed to hide the mini player

### DISCUSSION THREAD

**Discussion by Sreelekh-ABC** [LeoNatan/LNPopupController#501] [General] [answered]
Hi,
is theres any way to hide the mini player when we push to an another page.
In my case i only have to show this player where a tab bar is shown, on pages without tab bar the mini player is not needed.
using hidesBottomBarWhenPushed only hides the tab bar, the player is still visible.
I dont understand if this is something missing or something i misses.

Thank you Natan for this amazing library.

**Comment by LeoNatan** [comment, marked as answer]
Hello,
This used to be the behavior before the current one was implemented. I think, for the general case, the current one is preferred.
You can try dismissing and presenting the popup bar when you push/pop your controller that causes the tab bar to hide.
